docs/src/global/componentApi.jsx

```diff
@@ -0,0 +1,99 @@
+import {
+    Typography,
+} from 'react-cm-ui';
+import {
+    isEmpty,
+    map,
+} from 'lodash';
+// eslint-disable-next-line import/extensions
+import makeStyles from 'react-cm-ui/styles/makeStyles';
+import PropTypes from 'prop-types';
+import React from 'react';
+import TableProps from './tableProps';
+
+const propTypes = {
+    api: PropTypes.arrayOf(
+        PropTypes.shape({
+            heading: PropTypes.string,
+            props: PropTypes.arrayOf(PropTypes.shape({})),
+        }),
+    ),
+};
+
+const defaultProps = {
+    api: [],
+};
+
+const useStyles = makeStyles({
+    h2: {
+        marginTop: '33px',
+    },
+    h3: {
+        marginTop: '22px',
+    },
+    tableContainer: {
+        padding: '0 22px',
+    },
+});
+
+function ComponentApi(props) {
+    const {
+        api,
+    } = props;
+    const classes = useStyles();
+    const isApiEmpty = isEmpty(api);
+
+    if (isApiEmpty) {
+        return null;
+    }
+
+    let itemNumKey = 0;
+
+    return (
+        <div
+            className={classes.root}
+        >
+            <Typography
+                // anchor="api"
```

Comment:
Hope, we can remove the commented line. If no use.

Reply:
Yep, I will get that in a code clean up pass. Thanks.

Reply:
Fixed.

Reply:
I've confirmed this is fixed.  I'm gonna resolve this thread.